TimberAPI/SpecificationSystem/Fixes/CustomSpecifications/Golems/GolemPatch.cs

```diff
@@ -0,0 +1,20 @@
+using System.Linq;
+using HarmonyLib;
+using Timberborn.EntitySystem;
+using Timberborn.FactionSystemGame;
+using Timberborn.Golems;
+
+namespace TimberbornAPI.SpecificationSystem.Fixes.CustomSpecifications.Golems
+{
+    [HarmonyPatch(typeof(GolemFactory), "Load")]
+    public static class GolemFactoryPatch
+    {
+        static bool Prefix(ref Golem ____golemPrefab, ObjectCollectionService ____objectCollectionService,FactionService ____factionService)
```

Comment:
Why do these parameters have 4 underscores in them? AFAIK harmony doesn't require underscores to access method parameters?

Reply:
It's required for private parameters: https://harmony.pardeike.net/articles/patching-injections.html#___fields

Reply:
Ahh, gotcha